lib/kvdb/kvdb_home.c

```diff
@@ -42,7 +43,7 @@ path_join(const char *home, const char *path, char *buf, const size_t buf_sz)
         return 0;
     }
 
-    n = snprintf(buf, buf_sz, "%s%s%s", home, home[strlen(home)] == '/' ? "" : "/", path);
+    n = snprintf(buf, buf_sz, "%s%s%s", home, home[strlen(home) - 1] == '/' ? "" : "/", path);
```

Comment:
Presumably strlen(home) cannot be zero...

Reply:
Right. That's already been validated prior to this call